x/liquidity/client/cli/tx.go

```diff
@@ -42,16 +42,16 @@ func NewCreatePoolCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create-pool [pool-type-id] [deposit-coins]",
 		Args:  cobra.ExactArgs(2),
-		Short: "Create new liquidity pool with the specified pool type and deposit coins",
+		Short: "Create liquidity pool and deposit coins",
```

Comment:
That's valid point. We should take out the description about flags in the command and instead describe the flags in the command line help.

Reply:
@barriebyron Can you give us an example of which description of the flags should be described in the command line help?

Reply:
I had discussion with @dongsam about this issue offline, and this is the description of the flags in the command line help if we were to use flags instead. @barriebyron Please review the description and let us know If there is any issues with grammar, unclear sentences, or anything.  

```
1. NewCreatePoolCmd (I understand new and create mean the same but this is the code convention) 
- pool-type-id: The pool type id of the liquidity pool (the only supported pool type id is 1 in this version)
- deposit-coins: The deposit amount of coins to create pool (the number of deposit coins must be two in pool type 1)

2. NewDepositWithinBatchCmd
- pool-id: The pool id of the liquidity pool
- deposit-coins: The deposit amount of coins to the liquidity pool 

3. NewWithdrawWithinBatchCmd
- pool-id: The pool id of the liquidity pool
- pool-coin: The amount of pool coin to withdraw from the liquidity pool

4. NewSwapWithinBatchCmd
- pool-id: The pool id of the liquidity pool
- swap-type-id: The swap type id of the swap message (the only supported swap type is 1 in this version, which is instant swap)
- offer-coin: The amount of offer coin to swap
- demand-coin-denom: The denomination of the coin to exchange with offer coin
- order-price: The limit order price for the swap order, the price is the exchange ratio of X/Y where X is the amount of the first coin and Y is the amount of the second coin when their denoms are sorted alphabetically
- swap-fee-rate: The swap fee rate to pay for swap, which is proportional to swap amount. The swap fee rate must be the value that set as liquidity parameter in the current network.
```

Reply:
great, here's a few suggested changes. I'm not clear on when we want to use "number" versus "amount" 
```1. NewCreatePoolCmd 
- pool-type-id: The id of the liquidity pool. The only supported pool type id is 1. 
- deposit-coins: The number of coins to deposit to the liquidity pool. The number of deposit coins must be 2 in pool type 1.

2. NewDepositWithinBatchCmd 
- pool-id: The pool id of the liquidity pool 
- deposit-coins: The number of coins to deposit to the liquidity pool 

3. NewWithdrawWithinBatchCmd 
- pool-id: The pool id of the liquidity pool 
- pool-coin: The number of pool coins to withdraw from the liquidity pool 

4. NewSwapWithinBatchCmd 

- pool-id: The pool id of the liquidity pool 
- swap-type-id: The swap type of the swap message. The only supported swap type is 1 (instant swap).
- offer-coin: The amount of offer coin to swap 
- demand-coin-denom: The denomination of the coin to exchange with offer coin 
- order-price: The limit order price for the swap order. The price is the exchange ratio of X/Y where X is the amount of the first coin and Y is the amount of the second coin when their denoms are sorted alphabetically 
- swap-fee-rate: The swap fee rate to pay for swap that is proportional to swap amount. The swap fee rate must be the value that set as liquidity parameter in the current network.
```

Reply:
guidelines, pool-id is always pool-id and the description and usage should always be the same. It's human nature to want to explain the flag in context for each command, but the parameter is just a flag that always means the same thing, no matter the command it applies to. 
and so on

Reply:
Thank you for your suggestion
`amount` would be better for `deposit-coins` because the type is `sdk.Coins`

Reply:
I've been considering moving the arguments to Flags, and there's a problem that Flags is mixed with other global flags and ordering. 

Referring to usecase such as `gaiad tx ibc-transfer transfer [src-port] [src-channel] [receiver] [mount]`, we describe only help instead of add flag

Reply:
The concern is understandable. The changes will not be minor and they will confuse devs who are already used to using CLIs. We are going to have to consider this in another time.